packages/wallets/src/features/cashier/components/WalletCashierContent/WalletCashierContent.tsx

```diff
@@ -24,7 +28,14 @@ const WalletCashierContent = () => {
         }
     }, [isTransfer, isDeposit, isTransactions, isWithdraw, isResetBalance, isFiatOnRamp, history]);
 
-    if (isDeposit) return <WalletDeposit />;
+    if (isLoading) return <Loader />;
+
+    if (isDeposit && accountStatus)
```

Comment:
Do we need to check for accountStatus here?
```suggestion
    if (isDeposit)
```

Reply:
Updated implementation to check the account status in the respective components instead, as sending the API request from the different components with the same query shouldn't refetch the data from the API